drivers/counter/Kconfig.mcux_gpt

```diff
@@ -7,5 +7,6 @@ config COUNTER_MCUX_GPT
 	bool "MCUX GPT driver"
 	default y
 	depends on DT_HAS_NXP_IMX_GPT_ENABLED
+	select KERNEL_DIRECT_MAP if MMU
```

Comment:
but AFAICT using `DEVICE_MMIO_MAP` does not guarantee that the mapping is direct (1:1)

Reply:
May I know the reason? currently I am using:
`DEVICE_MMIO_MAP(dev, K_MEM_CACHE_NONE | K_MEM_DIRECT_MAP);`
so if CONFIG_MMU is enabled,  it will go into [z_phys_map()](https://github.com/zephyrproject-rtos/zephyr/blob/main/kernel/mmu.c#L752), so it will be 1:1 direct mapping.

Reply:
oooh, ok, I missed the `K_MEM_DIRECT_MAP` flag. nvm.